### PR TITLE
Feature/remove disclaimer from clinics view

### PR DESCRIPTION
**Pull request by Rallen13**
** Describe the functionality to be tested: **

- Removed disclaimer from clinics view.
- Updated date to October 5th, 2022 on about view.
- Updated tests for about view and clinics view.

** Giphy of how you feel submitting this PR: **

### CODE REVIEW

File: cypress/e2e/aboutview.cy.js [hunk 1]
```diff
@@ -66,7 +66,7 @@ describe("About", () => {
     cy.dataCy("data-statement")
       .should("be.visible")
       .contains(
-        "The data on this website was last updated on September 6, 2022."
+        "The data on this website was last updated on October 5, 2022."
```
Comment (lkarpins) [inline cypress/e2e/aboutview.cy.js:69]: Wooooo!!! Full state data!! Love this update!  💛

**Review by irmakerdem — APPROVED** [review]
Yay updated site! 🥳

File: src/Views/ClinicsView/ClinicsView.js [hunk 2]
```diff
@@ -53,15 +53,6 @@ const ClinicsView = () => {
       <div className="heading-container">
         <ClinicKeyBlock />
         <div>
-          <p
-            className="clinics-view-disclaimer"
-            data-cy="clinics-view-disclaimer"
-            tabIndex={0}
-          >
-            As this is an MVP project, we currently only have results for
```
Comment (lkarpins) [inline src/Views/ClinicsView/ClinicsView.js:61]: BIG LEAGUES! 😄

**Review by lkarpins — APPROVED** [review]
Looks great! Thanks for making those changes, Rachel!

**Pull request merged**